src/Kunstmaan/GeneratorBundle/Generator/KunstmaanGenerator.php

```diff
@@ -218,7 +218,12 @@ protected function generateEntityAdminType(
     protected function installDefaultPageTemplates($bundle)
     {
         // Configuration templates
-        $dirPath = sprintf('%s/Resources/config/pagetemplates/', $bundle->getPath());
+        if (Kernel::VERSION_ID >= 40000) {
+            $dirPath = sprintf('%s/../config/kunstmaancms/pagetemplates/', $bundle->getPath());
```

Comment:
```suggestion
                $dirPath = $this->container->getParameter('kernel.project_dir') . '/config/kunstmaancms/pagetemplates/';
```

To keep the code inline with the check in [`KunstmaanGenerateCommand`](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7b48b9c1d0bcecca7970011e8ef46d818d3a9ebb/src/Kunstmaan/GeneratorBundle/Command/KunstmaanGenerateCommand.php#L351)